tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+from tests import (  # noqa
```

Comment:
It seemed to be required as it was otherwise failing to execute the tests via the setup.py on python 3 (locally and on travis) without it.

It only complained about the locate_features tests, but I figured if we're putting one in we may as well put both.